jenkins/deploy.sh

```diff
@@ -103,12 +103,16 @@ $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
 SPARK2_PL=${SPARK2_PL:-"spark2-sql-plugin"}
 SPARK2_ART_ID=`mvn help:evaluate -q -pl $SPARK2_PL -Dexpression=project.artifactId -DforceStdout -Dbuildver=24X`
 SPARK2_ART_VER=`mvn help:evaluate -q -pl $SPARK2_PL -Dexpression=project.version -DforceStdout -Dbuildver=24X`
-SPARK2_FPATH="$M2DIR/repository/com/nvidia/$SPARK2_ART_ID/$SPARK2_ART_VER/$SPARK2_ART_ID-$SPARK2_ART_VER"
+SPARK2_FPATH="$M2DIR/com/nvidia/$SPARK2_ART_ID/$SPARK2_ART_VER/$SPARK2_ART_ID-$SPARK2_ART_VER"
```

Comment:
Do not have the subdir `repository` as we have the mvn option: [maven.repo.local](https://github.com/NVIDIA/spark-rapids/blob/branch-22.02/jenkins/spark-nightly-build.sh#L78)